mongo/client.go

```diff
@@ -329,6 +329,9 @@ func (c *Client) configure(opts *options.ClientOptions) error {
 		return opts.GetURI()
 	}))
 
+	// ClusterClock
+	c.clock = new(session.ClusterClock)
```

Comment:
This had to be moved up from its original location so we can reference `c.clock` in the handshakers we set.